xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -5152,3 +5152,8 @@ void CVideoPlayer::SetUpdateStreamDetails()
 {
   m_messenger.Put(new CDVDMsg(CDVDMsg::PLAYER_SET_UPDATE_STREAM_DETAILS));
 }
+
+void CVideoPlayer::TriggerCheckPassthrough()
+{
+  m_VideoPlayerAudio->SendMessage(new CDVDMsg(CDVDMsg::PLAYER_AUDIO_DEVICE_CHANGED), 1);
```

Comment:
this is wrong and can even segfault. m_VideoPlayerAudio can only be accessed from VP's internal thread

Reply:
You have a lot of knowledge of Kodi but that does not mean that everything you say is necessarily 100% true all the time.